From 7dc04fce45181f49b3448a377c599bbd254533a1 Mon Sep 17 00:00:00 2001
From: Ville Korhonen <ville@xd.fi>
Date: Wed, 5 Apr 2017 10:13:34 +0300
Subject: [PATCH] Use security mirror for current release

This replaces static release name 'jessie' with '{release}', that
gets replaced with current release name at build time.

Allows using security updates for current release, instead of using
those targeted for jessie.
---
 files/etc/apt/sources.list/CLOUD | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/files/etc/apt/sources.list/CLOUD b/files/etc/apt/sources.list/CLOUD
index f847cce..16f4420 100644
--- a/files/etc/apt/sources.list/CLOUD
+++ b/files/etc/apt/sources.list/CLOUD
@@ -1,3 +1,3 @@
 deb {cdn}/debian {release} main
-deb http://security.debian.org/ jessie/updates main
+deb http://security.debian.org/ {release}/updates main
 deb {cdn}/debian {release}-updates main
-- 
2.11.0

